### ISSUE TITLE
Fine Tune Pretrained YOLO11 Pose Model

### ISSUE THREAD

**Issue by joansc** [ultralytics/ultralytics#17198]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hello!

I would like to fine tune the YOLO11 pretrained Pose models on a dataset of 1k images. Is that possible? Would an nvidia 4090 be enough to handle the training? 

Thanks
Joan

**Comment by UltralyticsAssistant** [comment]
👋 Hello @joansc, thank you for your interest in Ultralytics 🚀! We're thrilled to see your enthusiasm for fine-tuning YOLO11 models. 

To get started, we recommend checking out our comprehensive [Docs](https://docs.ultralytics.com), which provide various [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples. These resources are valuable for both new users and those looking to dive deeper.

If this is a 🐛 Bug Report, you can help us by providing a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/). This will allow us to assist you more effectively.

For your custom training ❓ question, please share details such as dataset image examples and training logs. Ensuring that you're following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/) will also be beneficial.

Join our lively Ultralytics community on platforms that suit you best! For real-time chat, visit us on [Discord](https://ultralytics.com/discord) 🎧. If you prefer more detailed discussions, [Discourse](https://community.ultralytics.com) is a great place. Or explore discussions on our [Subreddit](https://reddit.com/r/ultralytics). We're excited to have you join the conversation!

## Upgrade

Make sure you're working with the latest version of the `ultralytics` package, ensuring all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) are met in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/). This is crucial to verifying your issue isn't already resolved:

```bash
pip install -U ultralytics
```

## Environments

YOLO is verified to run smoothly in several environments, complete with essential dependencies like [CUDA](https://developer.nvidia.com/cuda) and [PyTorch](https://pytorch.org/) preinstalled:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See our [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. Our [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/) can help set this up
- **Docker Image**. For Docker instructions, refer to our [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

A green Ultralytics CI badge indicates all tests are currently passing. We regularly test all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) across macOS, Windows, and Ubuntu.

This is an automated response, and an Ultralytics engineer will assist you soon. Feel free to explore the resources and community as you await further assistance 😊.

**Comment by Y-T-G** [comment]
Yes, it should be fine. Follow the docs.

https://docs.ultralytics.com/tasks/pose/

**Comment by joansc** [comment]
For testing purposes I created a dataset of segmented people of 130 images. I wanted to fine tune the yolo11x-seg model. For that I do the following:

```
model = YOLO("yolo11x-seg.pt")
results = model.train(data="custom_dataset/data.yaml", epochs=100, imgsz=640)
```
However, the exported model performs segmentation on people much worse than the pretrained one. I would expect that at least it would mantain the performance of the pretrained model... Any ideas?

**Comment by Y-T-G** [comment]
@joansc The performance in COCO would degrade since it's trying to maximize performance on your dataset.
